src/platform/datapath_winuser.c

```diff
@@ -565,7 +565,12 @@ QuicDataPathInitialize(
     WSADATA WsaData;
     QUIC_DATAPATH* Datapath;
     uint32_t DatapathLength;
+
     uint32_t MaxProcCount = QuicProcActiveCount();
+    QUIC_DBG_ASSERT(MaxProcCount <= UINT16_MAX);
+    if (MaxProcCount > UINT16_MAX) {
+        MaxProcCount = UINT16_MAX;
+    }
```

Comment:
Need to be 1 less then UINT16_MAX here. Otherwise all the loops with uint16_t control variables will infinite loop.

Reply:
Ah, good catch.